### DISCUSSION TITLE
Question about `tab` key completion

### DISCUSSION THREAD

**Discussion by Byxs20** [python-poetry/poetry#9083] [General] [answered]
Hi, I would like to know if `powershell` can have `tab` completion, looking at the documentation on the official website it doesn't seem to.

**Comment by Byxs20** [comment]
> Hi, I would like to know if `powershell` can have `tab` completion, looking at the documentation on the official website it doesn't seem to.

The bash I use is: oh-my-posh

**Comment by Secrus** [comment, marked as answer]
This is on the roadmap for our CLI framework and is not possible for now.
